scripts/check_envlogs.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python2.7
+"""Check the environment logs of many pexecs for similarity.
+
+Usage: check_envlogs.py <envlog dir>
+
+We check:
+ * The SHA1 hashes of the envlogs.
+ * The sizes of the envlogs.
+
+When hashing, we ignore `SUDO_COMMAND=` lines, since this contains a variable
+(but fixed-length) wrapper filename.
+
+The script prints nothing if all is well.
+"""
+
+import os
+import sys
+import hashlib
+from collections import defaultdict
+
+
+def hash_file(filename):
+    """Hash a file and return a hex string"""
+
+    hasher = hashlib.sha1()
+    with open(filename, 'rb') as fh:
+        for line in fh:
+            if line.startswith("SUDO_COMMAND="):
+                break  # the dash wrapper filename varies, we allow this
```

Comment:
Is it intentional to stop hashing at the first line in the file that starts with `SUDO_COMMAND`? That seems a bit odd to me.

Reply:
Maybe I can put it better: why do we care about stopping at `SUDO_COMMAND` at all?

Reply:
It's supposed to be `continue` oops!